app/views/YourSelfAssessmentCharges.scala.html

```diff
@@ -348,16 +393,60 @@ <h2 class="govuk-tabs__title">
+                                    <li class="govuk-tabs__list-item">
+                                        <a class="govuk-tabs__tab" href="#charges-due-in-30-days">
+                                            @getMessage("charges-due-in-30-days")
+                                        </a>
+                                    </li>
+                                    }
                                     <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
                                         <a class="govuk-tabs__tab" href="#placeholder">
                                             Placeholder
                                         </a>
                                     </li>
                                 </ul>
+
+                                @if(viewModel.whatYouOweChargesList.chargesDueWithin30DaysList.nonEmpty) {
+                                <div class="govuk-tabs__panel" id="charges-due-in-30-days">
+                                    <table class="govuk-table">
+                                        @tableHead("due-in-30-days-payments-table-head", false)
+                                        <tbody class="govuk-table__body">
+                                        @for((charge, index) <- ChargeItem.chargesDueWithin30DaysList(viewModel.whatYouOweChargesList).zipWithIndex) {
+                                        @tableRowWithoutInterest(s"due-30-days-$index", charge, index, "taxYearSummary-link")
+                                        }
+                                        <tr class="govuk-table__header">
+                                            <td class="govuk-table__header" colspan="2">
+                                                @getMessage("table.total-amount", viewModel.whatYouOweChargesList.balanceDetails.balanceDueWithin30Days.toCurrencyString)
+                                            </td>
+                                        </tr>
+                                        </tbody>
+                                    </table>
+
+                                    <div class="govuk-inset-text">
+                                        <div>
+                                            @getMessage("overdue-inset-text-1")
+                                        </div>
+                                        <br>
+                                        <div>
+                                            @getMessage("overdue-inset-text-2")
+                                        </div>
+                                    </div>
+
+                                    @if(user.saUtr.isDefined && !user.isAgent() && viewModel.whatYouOweChargesList.getEarliestTaxYearAndAmountByDueDate.isDefined) {
+                                    <div id="payment-button">
+                                        @link(link = controllers.routes.PaymentController.paymentHandoff(viewModel.whatYouOweChargesList.getEarliestTaxYearAndAmountByDueDate.get._2.toPence, origin).url,
```

Comment:
Please avoid use of .get and/o .head in the views when resolving data provided by viewModel;
I would suggest to add next property to the viewModel and throw and error on the viewModel construction phase:
`val earliestTaxYearAndAmountByDueDate : (Int, BigDecimal) `

+ _might worth to use TaxYear type instead of the Int?_